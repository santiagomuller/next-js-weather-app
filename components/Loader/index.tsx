import { FC, ReactElement } from 'react'

const Loader :FC = () :ReactElement => {
  return (
    <div className="loader h-full">Loading...</div>
  )
}

export default Loader