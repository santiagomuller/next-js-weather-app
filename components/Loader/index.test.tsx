import React from 'react';
import Loader from './index';
import renderer from 'react-test-renderer';

it('Loader renders without changes', () => {
  const tree = renderer.create(
    <Loader />
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});