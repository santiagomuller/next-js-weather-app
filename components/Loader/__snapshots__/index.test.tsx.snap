// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders without changes 1`] = `
<div
  className="loader h-full"
>
  Loading...
</div>
`;
